### PR TITLE
Python homework

### PR DESCRIPTION
**Pull request by temurbek-ismoilov**
Please check my homework
I could not complete tasks with *

### CODE REVIEW

File: 1.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/python
+print('ENTER YOUR NUMBERS:')
+x = list(input().split(' '))
```
Comment (nuclear0wl) [inline 1.py:3]: Will it be useful for the comma-separated numbers?

File: 2.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#! /usr/bin/python3.5
+
+numbers = [ 400, 401, 402, 403, 404, 234, 405, 406, 407 ]
+
+for x in numbers:
+    if x == 234:
+        print(x)
+        break;
+    elif x % 2 == 0:
```
Comment (nuclear0wl) [inline 2.py:9]: Just a tip: if you place this statement first (as if) and `if x == 234` as `elif` you will be able to remove excess if from the 7th string.

File: 1.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+print( x )
+print(x[1])
```
Comment (nuclear0wl) [inline 1.py:5]: Why is this required and where is a tuple?

File: 3.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/python3.5
+
+import sys
+
+x = list(sys.argv)
+
+for item in x[1:]:
+    print("ENTER YOUR WORD")
+    y = list(input().split(' '))
```
Comment (nuclear0wl) [inline 3.py:9]: Why these `split` and `list` are required? It breaks the support for phrases.

File: 3.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/python3.5
+
+import sys
+
+x = list(sys.argv)
+
+for item in x[1:]:
```
Comment (nuclear0wl) [inline 3.py:7]: This statement will exit the program after `len(x[1:])` tries even if the right word wasn't met.

File: 4.py [hunk 6]
```diff
@@ -0,0 +1,5 @@
+import os
+import pwd
+
+text = 'This script has the following PID: {ACTUAL_PID_HERE}. It was ran by {ACTUAL_USERNAME_HERE} to work happily on {ACTUAL_OS_NAME}-{ACTUAL_OS_RELEASE}'.format( ACTUAL_PID_HERE=str(os.getpid()), ACTUAL_USERNAME_HERE=pwd.getpwuid(os.getuid())[0], ACTUAL_OS_NAME=str(os.uname()[0]), ACTUAL_OS_RELEASE=str(os.uname()[2]))
```
Comment (nuclear0wl) [inline 4.py:4]: Does it show all the formatting types studied during the lecture?

**Comment by nuclear0wl** [comment]
Just a tip: try to put less debug or test info to the code that should be reviewed.

File: 1.py (hunk 1 above)
Comment (temurbek-ismoilov) [reply 1.py:3]: I'll fix it

File: 1.py (hunk 3 above)
Comment (temurbek-ismoilov) [reply 1.py]: Fixed!

File: 3.py (hunk 5 above)
Comment (temurbek-ismoilov) [reply 3.py:7]: I'll try to solve this task with 'while true'

File: 4.py (hunk 6 above)
Comment (temurbek-ismoilov) [reply 4.py:4]: My fault :(

Comment (temurbek-ismoilov) [reply 4.py]: Added other string formats

**Comment by nuclear0wl** [comment]
All is good! Thank you and sorry for the long review.

**Review by nuclear0wl — APPROVED** [review]
